src/components/SettingsDialog/Participants/Participants.scss

```diff
@@ -1,90 +1,232 @@
 @import "../SettingsDialog";
 
-.participants__container {
-  display: flex;
-  flex-direction: column;
-  height: auto;
-  margin-bottom: 62px;
-  overflow: auto;
-  flex-grow: 1;
+.participants__search-input-wrapper {
+  position: relative;
+  height: 40px;
+  width: 100%;
 }
-
-.participants__user-list-wrapper {
-  overflow: auto;
+.participants__search-input {
+  height: 100%;
   width: 100%;
-  // overflow-x: hidden; // firefox bug?
+  padding: 0 calc($padding--default + $icon--medium + $padding--small) 0 $padding--default;
+  font-size: 14px;
+  font-weight: bold;
+  letter-spacing: $letter-spacing--medium;
+  color: $color-black;
+  background-color: $color-white;
+  border: none;
+  border-radius: $rounded--full;
 
-  @include scrollbar();
+  &:focus {
+    outline: 2px solid var(--accent-color);
+  }
+  &:hover {
+    background-color: var(--accent-color--desaturated-light);
+  }
+}
+.participants__search-icon {
+  position: absolute;
+  height: 100%;
+  width: $icon--medium;
+  color: $color-black;
+  right: $padding--default;
 }
 
-.participants__user-list {
+.participants__filter-buttons {
+  margin-top: $margin--small;
   display: flex;
-  flex-direction: column;
-  width: 100%;
-  border-radius: 8px;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.participants__permisson-filter-button {
+  height: 42px;
   background-color: $color-white;
+  color: $color-black;
+  border: none;
+  border-radius: $rounded--full;
+  padding: 0 10px;
+  font-weight: bold;
+  font-size: 14px;
+  letter-spacing: $letter-spacing--medium;
+  cursor: pointer;
+
+  &--active {
+    background-color: var(--accent-color);
+    color: $color-white;
+  }
+  &:hover {
+    background-color: var(--accent-color--desaturated-light);
+    color: $color-black;
+  }
+  &:focus {
+    outline: 2px solid var(--accent-color);
+  }
+}
+@media screen and (min-width: 920px) {
+  .participants__permisson-filter-button {
+    padding: 0 $padding--default;
+  }
 }
 
-[theme="dark"] {
-  .participants__user-list {
-    background-color: $color-dark-mode-note;
+.participant__status-filter-button {
+  height: 42px;
+  width: 42px;
+  border: none;
+  border-radius: $rounded--full;
+  background-color: $color-white;
+  color: $color-black;
+  padding: 0;
+  cursor: pointer;
+
+  &--active {
+    background-color: var(--accent-color);
+    color: $color-white;
+  }
+  &:hover {
+    background-color: var(--accent-color--desaturated-light);
+    color: $color-black;
+  }
+  &:focus {
+    outline: 2px solid var(--accent-color);
+  }
+  > svg {
+    height: $icon--medium;
+    width: $icon--medium;
   }
 }
 
-.participants__user {
-  position: relative;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  padding: 0px $padding--medium;
-  height: 48px;
+.participants__list {
+  margin-top: $margin--default;
+  border-radius: $rounded--default;
+  list-style-type: none;
+  padding: 0;
+  overflow: scroll;
+  margin-bottom: 70px;
```

Comment:
`margin-bottom` in the Profile Settings is set by `$settings-dialog-container--bottom` (62px). We should stay consistent over the separate settings and use it here too. You are welcome to update the `$settings-dialog-container--bottom` to 70px if you want to.

Reply:
Thank you for pointing out, I'm going to use `$settings-dialog-container--bottom`